examples/python/testFixture.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python3
```

Comment:
Let's add a short README do this directory explaining how to run the example. Just for people who land on the directory directly and don't know about the tutorial.